src/phasorpy/components.py

```diff
@@ -311,3 +312,123 @@ def graphical_component_analysis(
             counts.append(numpy.asarray(component_counts))
 
     return tuple(counts)
+
+
+def components_unmixing_from_phasor(
+    multi_harmonic_real: ArrayLike,
+    multi_harmonic_imag: ArrayLike,
+    matrixA: ArrayLike,
+    /,
+) -> tuple[NDArray[Any]]:
+    """Return fractions in each pixel from multiple components.
+
+    Parametres
+    ----------
+    multi_harmonic_real : array_like
+        Real components of the phasor coordinate for many harmonics.
+    multi_harmonic_imag : array_like
+        Imaginary components of the phasor coordinate for many harmonics.
+    matrixA : array_like
+        Coefficiency matrix for each components.
+
+    Returns
+    -------
+    fractions : ndarray
+        Fractions of each components.
+
+    Raises
+    ------
+    ValueError
+        If multi_harmonic_real and multi_harmonic_imag have different shape.
+        If coefficiency matrix is empty.
+
```

Comment:
Add a reference to publication :ref:`Vallmitjana et al 2022 <vallmitjana-2022>`?